NewForce-Cohort-9/newforce-c9-tabloidmvc-TabloidMVC
# View All Tags #13

**Description**

I have completed the ticket to view all tags. In doing so I have created a tag management link in the navbar, so when the admin selects it, it directs them to the page with all of the tags, where they are able to see the list of all available tags in alphabetical order.

Trello Ticket: [https://trello.com/c/WAMiCprG](url)

Type of Change: New Feature

**Testing Instructions**
After making sure you have our repo cloned down into a local repository and have the database added, git fetch --all in the master branch to retrieve this branch named abby-viewalltag-13 that I have pushed up.

git checkout abby-viewalltag-13 to switch to my branch and open up Visual Studio

Then, run the application in a browser window. You should see the 'Home' page with a navigation bar at the top of the page. If not already logged out, log out (with the link in the navbar), and 'Log in' with the following credentials.
Username: [admin@example.com]

Once you're logged in, you should see the nav bar at the top left, with a new option to select 'Tag Management'. Click on this,
You will be directed to a "Tag Management" where you will see a list of all available tags in alphabetical order. No buttons or features will be functional on this page, it is simply just to view.

**Checklist:**
- [ ] I have opened abby-viewalltag-13 in my local repository 
- [ ]  I have ran my application in a browser window to see the 'Tag Management' on the navbar
- [ ]  After logging in the appropriate credentials, [admin@example.com], and clicking on 'Tag Management' in the navbar, it directs me to the page where you can view all of the available tags in alphabetical order
- [ ]  I have added test instructions that can be followed to test this code, to view this new feature.
